src/autolabel/transforms/pdf.py

```diff
@@ -0,0 +1,102 @@
+from typing import List, Dict, Any
+
+from autolabel.transforms import BaseTransform
+
+
+class PDFTransform(BaseTransform):
+    def __init__(
+        self,
+        output_columns: List[str],
+        file_path_column: str,
+        ocr_enabled: bool = False,
+        page_header: str = "Page {page_num}: ",
+        page_sep: str = "\n\n",
+    ) -> None:
+        """The output columns for this class should be in the order: [content_column, num_pages_column]"""
+        super().__init__(output_columns)
+        self.file_path_column = file_path_column
+        self.ocr_enabled = ocr_enabled
+        self.page_header = page_header
+        self.page_sep = page_sep
+
+        if self.ocr_enabled:
```

Comment:
If these are relatively disjoint considering if these should be put in separate classes as two separate PDFTransforms. This could make sense if we are planning to add more kinds of things like table reading to the pdf transform

@nihit @yadavsahil197 what do you think

Reply:
hmm, i would vote to keep it in the same one - easier for users to think of one transform for anything related to PDF loading - wdyt ?

Reply:
sg